src/types.d.ts

```diff
@@ -21,8 +21,12 @@ export interface Device {
   device_type: string
   openpilot_version: string
   sim_id: string
+  // not populated by api
+  fetched_at: number
```

Comment:
isn't this redundant with `WithFetchedAt`?

Reply:
definitely now with fetcher returning this for all cases